backend/FwHeadless/Services/SyncHostedService.cs

```diff
@@ -76,6 +78,16 @@ public bool QueueJob(Guid projectId)
 
         return addedToQueue;
     }
+
+    private void CacheRecentSyncResult(Guid projectId, SyncJobResult result)
+    {
+        memoryCache.Set($"SyncResult|{projectId}", result, TimeSpan.FromSeconds(5));
```

Comment:
Let's make this longer, like 30 seconds. If someone has a slow connection or drop or something weird it could take a bit to try to fetch the result again. That might be weird if someone triggers another sync, but I guess we should clear the cache to avoid that case.

Reply:
I'll increase it to 30s 👍

I don't quite understand the rest of your comment.
Currently the cache only gets checked if there is no job queued.
It's a fallback. As soon as someone triggers another sync, a previously cached result becomes inaccessible.
So I don't think it's necessary to clear it.

We can't really reverse the roles i.e. check the cache first, because in that case we'd **have** to clear the cache when a new sync starts and then we'd essentially just be back where we started, but with slightly more complicateed code.

Reply:
that's true, I think I forgot about that when I wrote that comment.